dolphinscheduler-ui/src/views/resource/components/resource/folder/use-form.ts

```diff
@@ -18,10 +18,11 @@
 import { reactive, ref, unref } from 'vue'
 import { useI18n } from 'vue-i18n'
 import type { FormRules } from 'naive-ui'
+import { ResourceType } from "@/views/resource/components/resource/types";
 
 const defaultValue = () => ({
   pid: -1,
-  type: 'FILE',
+  type: undefined as unknown as ResourceType,
   name: '',
   description: '',
   currentDir: '/'
```

Comment:
It is better to define an interface for the defaultValue.

Reply:
resolved